tools/generate-kaduna-prod-apk.sh

```
@@ -0,0 +1,20 @@
+cd apps/health_campaign_field_worker_app || exit
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)